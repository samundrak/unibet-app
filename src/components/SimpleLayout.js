import React from 'react';
import LayoutSkeleton from './LayoutSkeleton';
import Footer from './Footer';
import Header from './Header';
import Content from './Content';

const SimpleLayout = () => {
  return (
    <LayoutSkeleton
      header={() => <Header />}
      footer={() => <Footer />}
      navbar={() => <div />}
      content={() => <Content />}
    />
  );
};
export default SimpleLayout;
