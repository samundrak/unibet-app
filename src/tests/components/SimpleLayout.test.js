import React from 'react';
import { shallow } from 'enzyme';
import SimpleLayout from '../../components/SimpleLayout';

describe('Test SimpleLayout', () => {
  it('should test snapshot', () => {
    const wrapper = shallow(
      <SimpleLayout
        wasErrorFetchingLiveScore={false}
        loading={false}
        totalRecords={1}
        records={new Map([[1, {}]])}
      />,
    );
    expect(wrapper.html()).toMatchSnapshot();
  });
});
