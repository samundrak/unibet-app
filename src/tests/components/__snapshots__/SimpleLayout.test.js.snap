// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SimpleLayout should test snapshot 1`] = `"<div id=\\"container\\"><header><div id=\\"logo\\">Unibet</div></header><div></div><div id=\\"content\\"><article><h1>Live matches</h1><p class=\\"preamble\\">Here is a list of matches that are live right now.</p><aside><h2>Live betting</h2><p>Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.</p><p>You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.</p></aside><div id=\\"live-matches\\"><div width=\\"600px\\" height=\\"300px\\" class=\\"sc-bdVaJa hemWUb\\"><div class=\\"sc-bxivhb keacqw\\"></div></div></div></article></div><footer><div class=\\"inner\\"><p>Â© 1997-2015, Unibet. All rights reserved.</p></div></footer></div>"`;
